it/uk/gov/hmrc/soletraderidentificationfrontend/controllers/CheckYourAnswersControllerISpec.scala

```diff
@@ -506,6 +523,7 @@ class CheckYourAnswersControllerISpec extends ComponentSpecHelper
           stubStoreBusinessVerificationStatus(testJourneyId, BusinessVerificationNotEnoughInformationToCallBV)(OK)
           stubStoreRegistrationStatus(testJourneyId, RegistrationNotCalled)(OK)
           stubAudit()
+          stubRetrieveIndividualDetails(testJourneyId)(OK, testIndividualDetailsJsonNoNinoNoSautr ++ Json.obj("identifiersMatch" -> false))
```

Comment:
There is no scenario declared here, but the test is still working.

Reply:
Richard that it is a working in progress ... the presence of scenario it is just to point out that you need it .... the test is working because .... verifyAudit() is not doing anything behind the scene. The presence of the scenario avoid an exception during the auditing.... this file is 1045 lines ... I could not fix all the tests in this story.